core/wasm/build.rs

```diff
@@ -0,0 +1,5 @@
+//! Build script
+
+fn main() {
+    println!("cargo:rustc-link-arg=-zstack-size=2097152"); // 2 MiB
```

Comment:
I'd add an explanation for why we need this. IMO this variable needs fine-tuning. 2MiB is the lower end of what's acceptable for Rust. For context, the default on Linux is 8MiB. 

Since most of our work is loading the (ridiculously large) ISI and Queries, alongside the stack needed for the WASM dylib, we might need to adjust this number up.

Reply:
wasm's default is 1MB so I just doubled the value. But it can be tuned to our needs